pages/common/view.md

```diff
@@ -0,0 +1,8 @@
+# view
+
+> A read-only version of [vim](vim.md).
```

Comment:
Links in markdown like this won't be rendered correctly in the terminal. Perhaps we could rework this to include a `see also` on the second line?